pyuvsim/simsetup.py

```diff
@@ -719,10 +753,10 @@ def parse_frequency_params(freq_params):
         dict of array properties:
             |  channel_width: (float) Frequency channel spacing in Hz
             |  Nfreqs: (int) Number of frequencies
-            |  freq_array: (dtype float, ndarray, shape=(Nspws, Nfreqs)) Frequency channel centers in Hz
+            |  freq_array: (dtype float, ndarray, shape=(Nspws, Nfreqs)) Frequency channel
+            centers in Hz
```

Comment:
This is rendered very confusingly on RTD. I think the solution is to add the following to docs/conf.py:
```# set this to handle returns section more uniformly
napoleon_use_rtype = False
```

Reply:
Good idea, I've set those.